tests/unit/spec/base/mozilla-cookie-helper.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+/* For reference read the Jasmine and Sinon docs
+ * Jasmine docs: https://jasmine.github.io/
+ * Sinon docs: http://sinonjs.org/docs/
+ */
+
+/*
+|*|  * Mozilla.Cookies.removeItem(name[, path[, domain]])
+|*|  * Mozilla.Cookies.keys()
+*/
+
+describe('mozilla-cookie-helper.js', function () {
+    function clearCookies() {
+        document.cookies = '';
+    }
+
+    describe('setItem method', function () {
+        const cookieId = 'test-cookie';
+        var date = new Date();
+        date.setHours(date.getHours() + 48);
+
+        it('should set a cookie onto document.cookie', function () {
+            window.Mozilla.Cookies.setItem(cookieId, 'test', date, '/');
```

Comment:
Issue (blocking): we should make sure to clear cookies after every test here (tests run in a random order each run, so there's a risk not cleaning something up could lead to unexpected results).